src/components/manager/ManagerDropBoxContent.js

```diff
@@ -293,21 +294,44 @@ const ManagerDropBoxContent = () => {
   const ingestionWorkflows = workflowsByType.ingestion.items;
   const ingestionWorkflowsByID = workflowsByType.ingestion.itemsByID;
 
+  const [searchTerm, setSearchTerm] = useState("");
+
+  const handleSearchChange = (event) => {
+    setSearchTerm(event.target.value.toLowerCase());
+  };
+
   const filesByPath = useMemo(
     () => Object.fromEntries(recursivelyFlattenFileTree([], tree).map((f) => [f.relativePath, f])),
     [tree],
   );
 
-  const treeData = useMemo(
-    () => [
+  const filterTree = (nodes) => {
+    return nodes.reduce((acc, node) => {
```

Comment:
this should be a filter, not a reduce I think

Reply:
also, I think this function should be moved outside of the component, since it isn't a closure.

Reply:
I am not sure about filter,  could reduce be better here because the hierarchical structure of the folders?

Reply:
ah yes i see, its more of a filter-map. this works then.